File: IT/1/Variabili, operatori, strutture di controllo/Esercizi/Python/esercizio16.py
# file esercizio16.py

from __future__ import print_function
from sys import stdout
from os import linesep

m = 10
n = 10

for i in range(1, m+1):
	for j in range(1, n+1):
		print("{}\t".format(i * j), end="")
	stdout.write(linesep) 

File: IT/1/Variabili, operatori, strutture di controllo/Esercizi/Java/Esercizio16.java
/* file Esercizio16.java */

public class Esercizio16 {
	public static void main(String [] args) {
		final int M = 10;
		final int N = 10;

		for (int i = 1; i <= M; i++) {
			for (int j = 1; j <= N; j++) {
				System.out.print((i * j) + "\t");
			}

			System.out.print(System.lineSeparator());
		}
	}
}

